### PR TITLE
Senegal env data

### PR DESCRIPTION
**Pull request by CHelfgott**

### CODE REVIEW

File: routes/api/indicators.js [hunk 1]
```diff
@@ -61,13 +59,14 @@ router.get('/', authenticateResource('legends'), (req, res) => {
             dc.legend AS "legendText"
           FROM dataset_color AS dc
           WHERE dc.dataset_name = d.dataset_name
-          ORDER BY 2
+          ORDER BY 3
```
Comment (klandell) [inline routes/api/indicators.js]: Can you change this to be a field name

File: routes/api/queries/dhis/Network.js [hunk 2]
```diff
@@ -162,6 +168,119 @@ function getLevelStyles(data, organizationId) {
...
+      ed.fixture_id AS "fixtureId",
+      ed.average AS value,
+      p.temporal_res AS "temporalResolution"
+    FROM 
+       (SELECT product_id, dataset, fixture_id,
+          MAX(daymeasurement) AS max_day
+        FROM environmental_data
+        JOIN organization_datasets AS od2 USING (product_id, dataset)
+        WHERE od2.organization_id = ${organizationId}
+        GROUP BY 1,2,3) AS ed2
```
Comment (klandell) [inline routes/api/queries/dhis/Network.js]: Use column names

File: routes/api/indicators.js (hunk 1 above)
Comment (CHelfgott) [reply routes/api/indicators.js]: Done

File: routes/api/queries/dhis/Network.js (hunk 2 above)
Comment (CHelfgott) [reply routes/api/queries/dhis/Network.js]: Done

File: routes/api/queries/dhis/Timelines.js [hunk 3]
```diff
@@ -212,9 +219,54 @@ class IndicatorTimelines {
...
+          ed.daymeasurement AS "measDate",
+          ed.average AS value
+        FROM environmental_data AS ed
+        JOIN organization_datasets AS od USING (product_id, dataset)
+        JOIN dataset AS d USING (product_id)
+        WHERE od.organization_id = ${organizationId}
+        AND od.dataset = d.dataset_name
+        AND ed.fixture_id = ${nodeId}
+        AND d.dataset_label = ${indicatorNames[0]}
+        ORDER BY 1 DESC LIMIT 12
```
Comment (klandell) [inline routes/api/queries/dhis/Timelines.js]: @CHelfgott Can you update this one too.

Comment (CHelfgott) [reply routes/api/queries/dhis/Timelines.js]: Done

**Pull request merged**